tests/test_repo.py

```diff
@@ -87,13 +88,15 @@ def test_make_command_executes(self, cli_runner, test_clean_filesystem):
         assert not error_messages
 
 
+@pytest.mark.skip
 class TestRepoPython(BaseTestRepo):
     """Test scaffolding new python repository."""
 
     type_of_repo = "python"
     make_commands = "fmt", "lint", "test"
 
 
+@pytest.mark.skip
```

Comment:
Why skip this entire set of tests?

Reply:
This was temporary, since these tests were very poorly written and [caused interference with other tests](https://github.com/8ball030/auto_dev/pull/639#discussion_r2042973438). They have been re-enabled in PR #639, which was already merged into this PR, and this their state in this PR currently:
https://github.com/8ball030/auto_dev/blob/876757d1ef4521a3e4651313e31875051f6cd992/tests/test_repo.py#L1-L55